### PR TITLE
RCM-98:Odering list of drugs(Alphabetically) On Drug Order tab

### PR DESCRIPTION
**Pull request by HerbertYiga**
Ordered the list of drugs Alphabetically  on drug order tab.
I have attached the screen shorts of before and after on the ticket link.

Ticket link:    https://issues.openmrs.org/browse/RCM-98

### CODE REVIEW

**Comment by HerbertYiga** [comment]
@dkayiwa i am kindly requesting you to review this when you happen to get some time

File: omod/src/main/webapp/analysis/cohortBuilder.jsp [hunk 1]
```diff
@@ -912,7 +912,7 @@
 								<br/>
 								<select name="drugList" multiple="true" size="10">
 									<c:forEach var="drug" items="${model.drugs}">
-										<option value="${drug.drugId}"/>${drug.name}</option>
```
Comment (dkayiwa) [inline omod/src/main/webapp/analysis/cohortBuilder.jsp]: Why did you change from name to displayName?

Comment (mseaton) [reply omod/src/main/webapp/analysis/cohortBuilder.jsp]: @dkayiwa I assume this is just an up-port of what I already committed.  Display name was used because there are cases where name is null I believe.  Display name takes this into account.  Presumably the reason displayName exists is to provide a name for display :),

Comment (dkayiwa) [reply omod/src/main/webapp/analysis/cohortBuilder.jsp]: @mseaton in that case, do you think this is now ready for merging? :)

**Comment by mseaton** [comment]
LGTM - Merging
